core/repository/api/src/main/java/org/eclipse/rdf4j/repository/config/AbstractDelegatingRepositoryImplConfig.java

```diff
@@ -69,7 +72,11 @@ public Resource export(Model model) {
 
 		if (delegate != null) {
 			Resource delegateNode = delegate.export(model);
-			model.add(resource, CONFIG.delegate, delegateNode);
+			if (USE_CONFIG) {
+				model.add(resource, CONFIG.delegate, delegateNode);
+			} else {
+				model.add(resource, RepositoryConfigSchema.DELEGATE, delegateNode);
+			}
```

Comment:
The code is a bit bloated, but it should make it simpler to "revert" back to just using the CONFIG vocabulary.